packages/xsnap/test/test-inspect.js

```diff
@@ -56,26 +55,11 @@ const testCases = [
   ["new Proxy(() => {}, { get: () => 'foo' })", '[Function: foo]'],
 ];
 
-test('unconfined inspect', async t => {
-  for (const testCase of testCases) {
-    const [toEval, toRender] = Array.isArray(testCase)
-      ? testCase
-      : [testCase, testCase];
-    // eslint-disable-next-line no-eval
-    const evaled = (1, eval)(`(${toEval})`);
-    // t.log(evaled);
-    t.is(unconfinedInspect(evaled), toRender, toEval);
-  }
-});
-
 async function makeWorker() {
   const opts = options(io);
   const vat = xsnap(opts);
 
-  const boot = fs.readFileSync(
-    new URL('../dist/bundle-ses-boot.umd.js', import.meta.url).pathname,
-    'utf8',
-  );
+  const boot = fs.readFileSync(bundlePaths.sesBoot, 'utf8');
```

Comment:
Does `getLockdownBundle()` not work here? How about an analog for `sesBoot`? (yes, I'm asking again to get rid of `bundlePaths`).

Reply:
removed